tooling/lib/test/common.js

```diff
@@ -12,7 +12,6 @@ exports.gatherFiles = async function gatherFiles(env, packageName) {
 
   debug(`looking for ${env} tests for ${packageName}`);
   files = files.concat(await glob(`test/${env}/spec/**/*.[tj]s`, {packageName}));
-
```

Comment:
We can restore this change? This is unnecessary.

Reply:
Done